test/config/jest.image.ts

```diff
@@ -79,42 +79,41 @@ function saveAndRegisterImages(matcherContext: MatcherContext, received: Buffer,
     matchers: {}, // required by the jest-html-reporters getJestGlobalData function even if not used
   };
 
-  addAttach({
-    attach: computeRelativePathFromReportToSnapshots(`${baseImagePathWithName}-diff.png`),
-    description: 'diff',
-    bufferFormat: 'png',
-    context,
-  })
-    .then(() =>
-      addAttach({
-        attach: computeRelativePathFromReportToSnapshots(expectedImagePath),
-        description: 'expected',
-        bufferFormat: 'png',
-        context,
-      }),
-    )
-    .then(() => {
-      addAttach({
-        attach: computeRelativePathFromReportToSnapshots(receivedImagePath),
-        description: 'received',
-        bufferFormat: 'png',
-        context,
-      });
-    })
-    .catch(e =>
-      console.error(
-        `Error while attaching images to test ${snapshotIdentifier}.` +
-          `The 'jest-html-reporters' reporter is probably not in use. For instance, this occurs when running tests with the IntelliJ/Webstorm Jest runner.`,
-        e,
-      ),
+  try {
```

Comment:
**question:** why not keeping the promise chain and put a single await on top of the chain?

Reply:
Because for the inner `addAttach` callings, the error catching is missing. So ESLint failed with the `recommended-type-checked` rule :slightly\_frowning\_face: